src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpPrincipal.java

```diff
@@ -55,10 +55,11 @@ public HttpPrincipal(String username, String realm) {
      * and realm are equal to this object's username and realm. Returns {@code false}
      * otherwise.
      *
+     * @param another the object to compare this instance of {@code HttpPrincipal} against
```

Comment:
And that might require a CSR...

Reply:
I've created a CSR as requested: https://bugs.openjdk.java.net/browse/JDK-8255594

Reply:
CSR Reviewed.